src/patternfly/components/Dropdown/examples/dropdown-kebab-example.hbs

```diff
@@ -1 +1 @@
-{{#> dropdown id="dropdown-example-kebab" pf-c-dropdown--IsExpanded="true" pf-c-dropdown--Modifier="pf-m-plain" aria-label="Actions"}}<i class="fas fa-ellipsis-v"></i>{{/dropdown}}
\ No newline at end of file
+{{#> dropdown id="dropdown-example-kebab" pf-c-dropdown--IsExpanded="true" pf-c-dropdown--Modifier="pf-m-plain" aria-label="Actions"}}<i class="fas fa-ellipsis-v"  aria-hidden="true"></i>{{/dropdown}}
```

Comment:
I'd say for consistency let's pull the icon out of there and have a `pf-c-dropdown__kebab-icon="true"` and create a file for the icon.

Reply:
Agreed. The icon has been pulled out and placed in a file. pf-c-dropdown__kebab-icon="true" now pulls in the icon.